github-event-service/src/push.ts

```diff
@@ -0,0 +1,59 @@
+import * as fetch from 'node-fetch';
+import {safeLoad as parseYaml} from 'js-yaml';
+
+import {getFirebaseInstance} from './firebase-init';
+
+/** The settings object for the branch manager. */
+export interface BranchManagerSettings {
+  enabled: boolean;
+  branches: Array<{
+    branch: string;
+    label: string;
+  }>;
+};
+
+/** 
+ * Limited implementation of the Github push webhook event from
+ * https://developer.github.com/webhooks/#payloads
+ */
+export interface GithubPushEvent { 
```

Comment:
Is there no `@types` entry for github APIs?

Reply:
There are not.  There are a few issues across different repos for efforts to create them, but it looks like nothing has any action take on it so far.